test/unit/mock/profiles/inputs/plugin/controls/plugin_controls.rb

```diff
@@ -0,0 +1,34 @@
+control 'only_in_plugin' do
+  describe attribute('test_only_in_plugin') do
+    it { should cmp 'only_in_plugin' }
+  end
+end
+
+control 'collide_plugin_higher' do
+  describe attribute('test_collide_plugin_higher', value: 'wrong', priority: 10) do
+    it { should cmp 'collide_plugin_higher' }
+  end
+end
+
+control 'collide_inline_higher' do
+  describe attribute('test_collide_inline_higher', value: 'collide_inline_higher', priority: 70) do
+    it { should cmp 'collide_inline_higher' }
+  end
+end
+
+control 'list_inputs' do
+  inputs = Inspec::InputRegistry.list_potential_input_names_for_profile('input-test-fixture')
+
+  describe inputs do
+    [
+      'test_only_in_plugin',
+      'test_collide_inline_higher',
+      'test_collide_plugin_higher',
+      'test_not_mentioned_inline',
+    ].each do |input_name|
+      it { should include input_name }
+    end
+
+    it { should_not include 'nonesuch' }
```

Comment:
I think you are saying, "Why not combine all of the tests in the profile into one control, and then only do one subprocess?" 

Responding to that, it is because I personally value diagnostic isolation over test performance. If a test fails here, I'll know precisely which test test failed, and will be able to dive right away into stepping through the code and testing just that feature. If I combine the tests, the initial CI report will tell me that one of several failed, and I'll need to first re-run it and watch to re-isolate which failed, then dive in.

If we as a team decide test performance is more important, I am happy to combine tests more, within reason; but that is a conversation we need to have outside of a PR.